apps/funding-platform/src/blockchain/index.ts

```diff
@@ -0,0 +1,93 @@
+import { ethers, providers, Signer, Wallet } from "ethers";
+import { hardhatChainId } from "../utils";
+
+const productionChainIds: [number, ethers.utils.ConnectionInfo][] = [
+  [100, { url: "https://rpc.gnosischain.com/" }],
+];
+const developmentChainIds: [number, ethers.utils.ConnectionInfo][] = [
+  [hardhatChainId, { url: "http://localhost:8545" }],
+];
+export const chainIds = new Map<number, ethers.utils.ConnectionInfo>(
+  // eslint-disable-next-line turbo/no-undeclared-env-vars
+  process.env.NODE_ENV === "production"
+    ? productionChainIds
+    : developmentChainIds
+);
+
+export const sendTransaction = async (params: {
```

Comment:
since our queueing is sequential, we don't need to worry about passing the `nonce` of the transaction, however, this in the future will need to be updated